src/Bicep.LangServer/Handlers/BicepDeploymentParametersHandler.cs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
MUST FIX: You're writing values into the params file even if I say "no" to the prompt

Main.bicep:
 
param int1 int = 123
param int2 int
 
Main.params.json:
 
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {
    }
}
 
During deploy, enter "321" for both parameters.
Select "no" to saving
 
Main.params.json after deployment:
 
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {
      "int2": {
        "value": 432
      },
      "int1": {
        "value": 321
      }
    }
}

Reply:
I can't repro anymore, either.